src/Grid/Grid.js

```diff
@@ -190,7 +190,7 @@ type Props = {
    */
   gutter?: 0 | 8 | 16 | 24 | 40,
   /**
-   * If provided, will wrap with Hidden component and given properties.
+   * If provided, will wrap with [Hidden component] and given properties.
```

Comment:
I thought about that and it was my original approach, but wasn't sure if it would add too much noise to the comments.  I will go with this!  :+1: